test/e2e/e2e_test.go

```diff
@@ -76,6 +76,7 @@ func TestMain(m *testing.M) {
 	flag.StringVar(&globalOpts.configurableClusterOptions.SSHKeyFile, "e2e.ssh-key-file", "", "Path to a ssh public key")
 	flag.StringVar(&globalOpts.platformRaw, "e2e.platform", string(hyperv1.AWSPlatform), "The platform to use for the tests")
 	flag.StringVar(&globalOpts.configurableClusterOptions.NetworkType, "network-type", "", "The network type to use. If unset, will default based on the OCP version.")
+	flag.StringVar(&globalOpts.configurableClusterOptions.ServicePublishingStrategy, "service-publishing-strategy", "Ingress", "Define how to expose the cluster services. Supported options: Ingress (Use LoadBalancer and Route to expose services), NodePort (Select a random node to expose service access through) (default \"Ingress\")")
```

Comment:
can you call this command the `e2e.kubevirt-service-publishing-strategy`, and set the default value to the enum `kubevirt.IngressServicePublishingStrategy` 

My reasoning here is that the service publishing strategy appears to only be exposed as a KubeVirt option on the cli and not a platform wide option.

Reply:
done.